
Mark as 4.7 so we can rebuild aufs tools with matching 4.7 check

Signed-off-by: Thomas Backlund <tmb@mageia.org>

--- linux/include/uapi/linux/aufs_type.h.orig
+++ linux/include/uapi/linux/aufs_type.h
@@ -26,7 +26,7 @@
 
 #include <linux/limits.h>
 
-#define AUFS_VERSION	"4.x-rcN"
+#define AUFS_VERSION	"4.7"
 
 /* todo? move this to linux-2.6.19/include/magic.h */
 #define AUFS_SUPER_MAGIC	('a' << 24 | 'u' << 16 | 'f' << 8 | 's')
